lib/streamlit/config.py

```diff
@@ -293,6 +299,36 @@ def _section_memoized_option_name():
     return option
 
 
+def _create_theme_options(
+    name: str,
+    categories: list[Literal["theme"] | CustomThemeCategories],
+    description: str | None = None,
+    default_val: Any | None = None,
+    visibility: str = "visible",
+    type_: type = str,
+) -> None:
+    """
+    Create ConfigOption(s) for a theme-related config option and store it globally in this module.
+    The same config option can be supported for multiple categories, e.g. "theme" and "theme.sidebar".
+    """
+    for cat in categories:
+        section = cat if cat == "theme" else f"theme.{cat.value}"
+
+        _create_option(
+            f"{section}.{name}",
+            description,
+            default_val,
+            False,  # scriptable
+            visibility,
+            False,  # deprecated
+            None,  # deprecation_text
+            None,  # expiration_date
+            None,  # replaced_by
+            type_,
+            False,  # sensitive
```

Comment:
nit: Instead of adding these comments, I think it's better to just directly use keyword arguments to provide these parameters. This also makes it a lot more explicit.